dialect/sql/schema/postgres.go

```diff
@@ -214,16 +214,19 @@ const maxCharSize = 10 << 20
 // scanColumn scans the information a column from column description.
 func (d *Postgres) scanColumn(c *Column, rows *sql.Rows) error {
 	var (
-		nullable sql.NullString
-		defaults sql.NullString
+		nullable        sql.NullString
+		defaults        sql.NullString
+		userDefinedType sql.NullString
 	)
-	if err := rows.Scan(&c.Name, &c.typ, &nullable, &defaults); err != nil {
+	if err := rows.Scan(&c.Name, &c.typ, &nullable, &defaults, &userDefinedType); err != nil {
 		return fmt.Errorf("scanning column description: %v", err)
 	}
 	if nullable.Valid {
 		c.Nullable = nullable.String == "YES"
 	}
 	switch c.typ {
+	case "USER-DEFINED":
+		c.Type = field.TypeOther
```

Comment:
Don't you want to move it to line 255? Or you plan to add more logic here.

Reply:
nice catch, no extra logic for now